### PR TITLE
Adds action figure toys

### PR DESCRIPTION
**Pull request by ZomgPonies**
- Ports action figure toys from Hippie
https://github.com/HippieStationCode/HippieStation13/pull/1159

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
This is actually an NT station thing, not Hippie.

**Comment by Nienhaus** [comment]
This is actually an @Mrsnapwalk thing, not NT station.

**Comment by Spacemanspark** [comment]
Oh, I remember these.
:+1:

**Comment by KasparoVy** [comment]
Cute as hell, great sprites!

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1442,3 +1443,237 @@ obj/item/toy/cards/deck/syndicate/black
...
+/obj/item/toy/figure
+	name = "Non-Specific Action Figure action figure"
+	desc = "A \"Space Life\" brand... wait, what the hell is this thing?"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "nuketoy"
+	var/cooldown = 0
+	var/toysay = "What the fuck did you do?"
+
+/obj/item/toy/figure/attack_self(mob/user as mob)
+	if( cooldown < world.time )
```
Comment (tigercat2000) [inline code/game/objects/items/toys.dm]: >I need a formatting nazi title

this has spaces in the if
byond standard styling has no spaces in the if

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -1442,3 +1443,237 @@ obj/item/toy/cards/deck/syndicate/black
...
+	toysay = "Oh god, the singularity is loose!"
+
+/obj/item/toy/figure/geneticist
+	name = "Geneticist action figure"
+	desc = "A \"Space Life\" brand Geneticist action figure."
+	icon_state = "geneticist"
+	toysay = "I'm not qualified for this job."
+
+/obj/item/toy/figure/hop
+	name = "Head of Personel action figure"
```
Comment (tigercat2000) [inline code/game/objects/items/toys.dm]: [Personel](https://en.wikipedia.org/wiki/Personel)
[Personnel](http://www.merriam-webster.com/dictionary/personnel)

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -1442,3 +1443,237 @@ obj/item/toy/cards/deck/syndicate/black
...
+
+/obj/item/toy/figure/secofficer
+	name = "Security Officer action figure"
+	desc = "A \"Space Life\" brand Security Officer action figure."
+	icon_state = "secofficer"
+	toysay = "I am the law!"
+
+/obj/item/toy/figure/virologist
+	name = "Virologist action figure"
+	desc = "A \"Space Life\" brand Virologist action figure."
```
Comment (tigercat2000) [inline code/game/objects/items/toys.dm]: these descriptions literally all follow the exact same naming convention
they shouldn't be hardcoded at all, there should be something like a /figure/New() proc
which sets desc equal to
`desc = "A \"Space Life\" brand [name]"`

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -1442,3 +1443,237 @@ obj/item/toy/cards/deck/syndicate/black
...
+	name = "Chief Medical Officer action figure"
+	desc = "A \"Space Life\" brand Chief Medical Officer action figure."
+	icon_state = "cmo"
+	toysay = "Suit sensors!"
+
+/obj/item/toy/figure/assistant
+	name = "Assistant action figure"
+	desc = "A \"Space Life\" brand Assistant action figure."
+	icon_state = "assistant"
+	toysay = "Grey tide world wide!"
```
Comment (KasparoVy) [inline code/game/objects/items/toys.dm]: Grey tide station wide!

**Comment by Plotron** [comment]
The Captain looks off to me... Look at his face and then look at his hands.

I like the idea.

File: code/game/objects/items/toys.dm [hunk 5]
```diff
@@ -1442,3 +1443,205 @@ obj/item/toy/cards/deck/syndicate/black
...
+/obj/item/toy/figure
+	name = "Non-Specific Action Figure action figure"
+	desc = "A \"Space Life\" brand... wait, what the hell is this thing?"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "nuketoy"
+	var/cooldown = 0
+	var/toysay = "What the fuck did you do?"
+
+/obj/item/toy/figure/New()
+	desc = "A \"Space Life\" brand [name]"
```
Comment (tigercat2000) [inline code/game/objects/items/toys.dm]: No idea if there is something actually important in the parents of this, but it probably should call & return parent anyways.

**Pull request merged**